setup.py

```diff
@@ -30,7 +30,12 @@ def run(self):
     cmdclass = { 
         'install': PostInstallCommand,
     },
-    scripts = ['seoanalyzer/seoanalyze'],
+    # no idea why this needed fixing, the diff says it didn't change...
```

Comment:
You renamed the script seoanalyzer to __main__ with the main function; make sense.